### ISSUE TITLE
Unmatched image list and downloaded content

### ISSUE THREAD

**Issue by akallen** [hiroyasuakada/UnrealEgo#3]
I am interested in checking the implementation of this work. However, when I downloaded the folder from Google Drive, I found the downloaded content and the image list is not matching. For example, the 'rp_adanna_rigged_003_ue4' has following:
```
CityPark_Showcase
    └───Day
        ├───rp_adanna_rigged_003_ue4
        │   └───SKM_MenReadingGlasses_Shape_01
        │       ├───004
        │       │   └───Pulling_A_Rope__7_
        │       │       └───all_data_with_img-256_hm-64_pose-16_npy
        │       ├───019
        │       │   └───Tender_Placement__1_
        │       │       └───all_data_with_img-256_hm-64_pose-16_npy
        │       ├───022
        │       │   └───Left_Strafing_Jump__1_
        │       │       └───all_data_with_img-256_hm-64_pose-16_npy
        │       ├───023
        │       │   └───Silly_Dancing__10_
        │       │       └───all_data_with_img-256_hm-64_pose-16_npy
        │       ├───025
        │       │   ├───Chapa_2
        │       │   │   └───all_data_with_img-256_hm-64_pose-16_npy
        │       │   └───Smash__5_
        │       │       └───all_data_with_img-256_hm-64_pose-16_npy
        │       └───026
        │           ├───Soccer_Header
        │           │   └───all_data_with_img-256_hm-64_pose-16_npy
        │           └───Soccer_Trip
        │               └───all_data_with_img-256_hm-64_pose-16_npy
```
But the validation set has following:
```
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/019/Tender_Placement__1_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Swing_Dancing__18_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Swing_Dancing__26_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/024/Body_Block__12_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/026/Soccer_Header
```
Or the training set's list:
```
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/003/Back_Squat__1_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/003/Jump_Push_Up__1_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/004/Pulling_A_Rope__7_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/005/Guitar_Playing__3_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/005/Guitar_Playing__25_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/010/Looking_Through_Files_Low
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/017/Blow_A_Kiss
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/017/Opening__23_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/017/Shooting_Gun__4_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/021/Standing_Walk_Left
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/021/Walk_Backwards
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/022/Left_Strafing_Jump__1_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/022/Short_Left_Side_Step
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/022/Standing_Torch_Walk_Left__3_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Brooklyn_Uprock
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Hip_Hop_Dancing__6_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Silly_Dancing__10_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/023/Snake_Hip_Hop_Dance__4_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/024/Body_Block__3_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/024/Livershot_Knockdown__1_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/024/Taunt
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/025/Chapa_2
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/025/Smash__5_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/026/Goalkeeper_Sidestep__3_
../../../../../CT/UnrealEgo/static00/UnrealEgoData/CityPark_Showcase/Day/rp_adanna_rigged_003_ue4/SKM_MenReadingGlasses_Shape_01/026/Soccer_Trip
```
I would like to know does 'UnrealEgo impl' folder contain all necessary npy files for training or for validation? It seems like the file that we downloaded are missing some files for both cases. Just want to know if this issue only happens on my side. Thanks!

**Comment by hiroyasuakada** [comment]
Hi, thank you for your interest in our dataset! 

I was able to access all of the dataset from several Google accounts.

Also, as mentioned in the README, you need to modify the lists in the txt files based on your local paths, e.g., (local path)/UnrealEgoData/Lab_Scene/Day/rp_eric_rigged_001_ue4/SKM_MenReadingGlasses_Shape_01/026/Chip.

Here, you don’t need to add /all_data_with_img-256_hm-64_pose-16_npy to the paths.

**Comment by akallen** [comment]
Thank you for your quick response. I understand that I need to modify the prefix and no need to add the _npy path at the end of the string. However, the issue is that some of my downloaded folders are missing according to the folder list.

May I know what files should be contained in the 'UnrealEgo impl'? Will that cover all the folders listed in training / validation / testing?

I also tried to download both UnrealEgo 02's and UnrealEgoData-impl-01's CityPark_Showcase folders and combined them together, the result is also different:

```
CityPark_Showcase
└───Day
    ├───rp_adanna_rigged_003_ue4
    │   └───SKM_MenReadingGlasses_Shape_01
    │       ├───004
    │       │   └───Pulling_A_Rope__7_
    │       │       └───json
    │       ├───017
    │       │   ├───Opening__23_
    │       │   │   ├───fisheye_depth_image
    │       │   │   │   ├───camera_left
    │       │   │   │   └───camera_right
    │       │   │   └───fisheye_final_image
    │       │   │       └───camera_right
    │       │   └───Shooting_Gun__4_
    │       │       ├───all_data_with_img-256_hm-64_pose-16_npy
    │       │       └───fisheye_depth_image
    │       │           └───camera_right
    │       ├───021
    │       │   └───Standing_Walk_Left
    │       │       └───fisheye_depth_image
    │       │           └───camera_right
    │       ├───023
    │       │   ├───Swing_Dancing__18_
    │       │   │   └───fisheye_depth_image
    │       │   │       └───camera_right
    │       │   └───Swing_Dancing__26_
    │       │       ├───fisheye_depth_image
    │       │       │   ├───camera_left
    │       │       │   └───camera_right
    │       │       └───json
    │       ├───025
    │       │   └───Smash__5_
    │       │       └───fisheye_depth_image
    │       │           └───camera_left
    │       └───026
    │           ├───Soccer_Header
    │           │   └───all_data_with_img-256_hm-64_pose-16_npy
    │           └───Soccer_Trip
    │               └───all_data_with_img-256_hm-64_pose-16_npy
```

Very weird result it seems.

**Comment by hiroyasuakada** [comment]
Can you please check if the missing dataset is available on Google Drive at first?

Also, are you using wget or gdrive? I realized that they can be used to download a small number of files (up to 50 files or so) so some of the data will be missing.

So, I will split the dataset to a smaller number of zip files so that users can use wget, for instance.

For now, you can download the dataset directly from Google Drive or write some code to access the dataset.

**Comment by akallen** [comment]
Those missing files are avaible on the Google Drive website, but when I tried to download them, I guess something went wrong in the Google Drive's zipping function.

It seems to be a known issue for Google Drive: https://support.google.com/drive/thread/4142625/download-all-doesn-t-capture-all-files?hl=en 

Maybe could you directly upload zip files so that we can download them directly to avoid this bug? Thank you!

**Comment by hiroyasuakada** [comment]
Apparently, this bug hasn’t been solved for a long time :)

I’ll upload the zip version as soon as possible.

**Comment by hiroyasuakada** [comment]
Hi, I uploaded the zip version of "UnrealEgo impl 1~3". You can just download them to try our implementation now.

I'm also uploading the zip version of other datasets (UnrealEgo 1~6) now so stay tuned.

Please let me know if you encounter any issues.
